### ISSUE TITLE
Add day or night flag

### ISSUE THREAD

**Issue by djsegal** [chubin/wttr.in#235]
Don't know if this is the correct place to suggest it, but a quick day/night flag might be interesting?

You already have their area code, so you would just need to check it against the current world map sunrise/sundown timer?

**Comment by chubin** [comment]
Yes, it is a good idea, and we are implementing already something like that for the oneline format (#174).
@djsegal Den, would it be a solution for you?

**Comment by djsegal** [comment]
That is pretty interesting! Would there be a mode that just checks day/night though?

// feel free to close this

**Comment by chubin** [comment]
Yes, we can add it. But how exactly it should look like?
Just output `day/night`? Is it that?

**Comment by djsegal** [comment]
Thats probably the best way to do it?

You could use day and night emojis (☀🌙)

// or if you want to go the extra mile -- what moon it is in the lunar cycle, day in the solar cycle?

**Comment by ChazyTheBest** [comment]
Hi, any news about this?

Sitting in front of my computer monitor and seeing a Sun in the middle of the night is a bit weird.

Using `format=1`.

**Comment by chubin** [comment]
@ChazyTheBest Maybe you mean this one: #52 ?
It is not yet done also.

But thank you for mentioning this issue, because in the v2 view we already have the
feature, that @djsegal originally wanted.

@djsegal Please try `curl v2.wttr.in`. Is it that?

**Comment by ChazyTheBest** [comment]
@chubin Yes, sorry, I didn't search enough. Thanks, I'll follow issue #52.
